docs/reporting.rst

```diff
@@ -0,0 +1,164 @@
+#####################
+ICA Components Report
+#####################
+
+The reporting page for the tedana decomposition presents a series
+of interactive plots designed to help you evaluate the quality of your
+analyses. This page describes the plots forming the reports and well as
+infromation on how to take advantage of the interactive functionalities.
+You can also play around with `our demo`_.
+
+.. _our demo: https://me-ica.github.io/tedana-ohbm-2020/
+
+Report Structure
+================
+
+The image below shows a representative report, which has two sections: a) the summary view,
+and b) the individual component view.
+
+.. image:: /_static/rep01_overallview.png
```

Comment:
I don't think this reflects the state of the reports any more, unfortunately, since we removed the individual component view. The figure should be updated

Reply:
[1b59407](https://github.com/ME-ICA/tedana/pull/465/commits/1b594074fdfa0fcc48f6cd28caf4fd3e397b13c3) should take care of this.

I included the individual component view section in the end, as we do show a static view of individual components.

Reply:
That looks right, thanks !! Should we note that the individual component view will be empty by default, since no component is pre-selected ?